Gaijinizer patch/Map003.txt

```diff
@@ -1327,10 +1326,10 @@ there.#MTLed
 \\C[2]後半になるにつれて、知識を吸い取られて不利になる。
 回復とかは、前半に済ませておくのが吉だぜ。
 > CONTEXT: Map003/events/1/pages/2/318/Dialogue/クッション君
-Just watch out? \\C[2]As we get to the second half, we
-lose knowledge and become disadvantaged.
-For recovery and recovery, it is good to finish
-in the first half.#MTLed
+Just... be careful. \\C[2]As you get into the second
+half of the battle, your drained intelligence
+will be a real handicap. It's a good idea to use
+your recovery items and skills in the first half.#RonIffy
 > END STRING
```

Comment:
Minor bit of flow (this might be a localization):
>_**It's a good idea to use
your recovery items and skills while you still can.**_